types/config.go

```diff
@@ -85,11 +85,13 @@ func (config *Config) SetAddressVerifier(addressVerifier func([]byte) error) {
 	config.addressVerifier = addressVerifier
 }
 
+// TODO
 func (config *Config) SetCoinType(coinType uint32) {
 	config.assertNotSealed()
 	config.coinType = coinType
 }
 
+// TODO
```

Comment:
maybe it is a good idea to complete this now.. otherwise it will be left there for ages..

Reply:
added